pymeasure/display/windows.py

```diff
@@ -681,10 +713,54 @@ class ManagedImageWindow(ManagedWindow):
     def __init__(self, procedure_class, x_axis, y_axis, z_axis=None, **kwargs):
         self.z_axis = z_axis
         self.image_widget = ImageWidget(
-            "Image", procedure_class.DATA_COLUMNS, x_axis, y_axis, z_axis)
+            "Image", procedure_class.DATA_COLUMNS, x_axis, y_axis, z_axis
+        )
 
         if "widget_list" not in kwargs:
             kwargs["widget_list"] = ()
-        kwargs["widget_list"] = kwargs["widget_list"] + (self.image_widget, )
+        kwargs["widget_list"] = kwargs["widget_list"] + (self.image_widget,)
 
         super().__init__(procedure_class, x_axis=x_axis, y_axis=y_axis, **kwargs)
+
+
+class InstrumentControlWindow(QtGui.QMainWindow):
```

Comment:
I'm sure you plan to do this in a subsequent commit, but adding docstrings to this class in a manner similar to the ManagedWindow classes should definitely be added.

Reply:
Of course! Did not do it yet because the structure might still change